splunklib/client.py

```diff
@@ -463,6 +463,13 @@ def info(self):
         response = self.get("/services/server/info")
         return _filter_content(_load_atom(response, MATCH_ENTRY_CONTENT))
 
+    def input(self, path, kind=None):
```

Comment:
Not a big deal, but should we follow Go style practices of adding a new prefix to the declaration?

Like this
`def new_input(self, path, kind=None):`.

Reply:
not if we want consistency, so I'm going to say let's not do that here